packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/add/translations/Messages_de_DE.json

```diff
@@ -1,7 +1,6 @@
 {
-  "add_an_account": "Ajouter un compte xDSL E-mail",
```

Comment:
Following key `add_an_account` is removed because it's not present in the source language file, isn't it?

I mean: `packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/add/translations/Messages_fr_FR.json`

Reply:
@antleblanc, @marie-j

The tool compare source language translations keys with target language translations keys in the same directory.

`add_an_account` isn't present in `fr_FR` file, so it will be deleted from this target file.